src/main/java/com/google/api/generator/gapic/composer/comment/CommentComposer.java

```diff
@@ -20,7 +20,7 @@
 
 public class CommentComposer {
   private static final String APACHE_LICENSE_STRING =
-      "Copyright 2021 Google LLC\n\n"
+      "Copyright 2022 Google LLC\n\n"
```

Comment:
Can we just put in the current year, so we don't have to update it every year?

Reply:
I thought about it, the concern is that our golden tests would start failing all of sudden in the beginning of next year, which is not a good behavior in my opinion. I can update it to use current year if this behavior is OK.